src/main/java/org/folio/bulkops/service/BulkOperationService.java

```diff
@@ -191,9 +196,13 @@ public BulkOperation uploadCsvFile(EntityType entityType, IdentifierType identif
       try {
         var linkToTriggeringFile = remoteFileSystemClient.put(multipartFile.getInputStream(), operation.getId() + "/" + multipartFile.getOriginalFilename());
         operation.setLinkToTriggeringCsvFile(linkToTriggeringFile);
-      } catch (Exception e) {
-        log.error(ERROR_STARTING_BULK_OPERATION + e);
+      } catch (S3ClientException e) {
         errorMessage = ERROR_NOT_UPLOAD_FILE_S3_INVALID_CONFIGURATION;
+      } catch (Exception e) {
+        errorMessage = format(FILE_UPLOADING_FAILED_REASON, e.getMessage());
+      }
+      if (nonNull(errorMessage)) {
```

Comment:
Handling of error message outside catch-block makes code dirty. Please, make error messages logging for particular exception inside corresponding catch block.